doc/man/pwquality.conf.5.pod

```diff
@@ -111,6 +111,12 @@ If nonzero, check whether the password (with possible modifications)
 contains the user name in some form. It is not performed for user names shorter
 than 3 characters. (default 1)
 
+=item B<usersubstr=>I<N>
+
+If greater than 3 (due to the minimum length in usercheck), check whether the
+password contains a substring of at least usersubstr length in some form.
```

Comment:
`...the password contains a substring of the username of at least I<N> length in some form.`

Reply:
`s/usersubstr/I<N>/`